kapitan/inputs/jsonnet.py

```diff
@@ -70,7 +70,7 @@ def compile_file(self, file_path, compile_path, ext_vars, **kwargs):
         ext_vars will be passed as parameters to jsonnet_file()
         kwargs:
             output: default 'yaml', accepts 'json'
-            prune: default False, accepts True
+            prune_output: default False, accepts True
```

Comment:
this is a bit confusing. Aren't these changes part of https://github.com/kapicorp/kapitan/pull/999 and not related to remote fetching?

Reply:
this commit has gone once I've done the rebase